src/dataSources/cloudFirestore/session.js

```diff
@@ -8,30 +8,44 @@ const sessionDateForge = utility.firestoreDateForge.sessions;
 const collectionName = 'sessions';
 const approvedSessionStatuses = ['ACCEPTED', 'SCHEDULED', 'CANCELLED'];
 
+function validateStatuses(statuses) {
+  dlog('validateStatuses %o', statuses);
+  if (!Array.isArray(statuses) || statuses.length === 0) {
+    throw new Error('statuses must be in the form of an array with a value.');
+  }
+  const inStatus = statuses;
+  const isidx = inStatus.indexOf('APPROVED');
```

Comment:
I assume you mena isIndex.. I'd just spell it out to create from causing confusion.